src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Security/index.html.twig

```diff
@@ -0,0 +1,65 @@
+{#**
+  * Copyright since 2007 PrestaShop SA and Contributors
+  * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+  *
+  * NOTICE OF LICENSE
+  *
+  * This source file is subject to the Open Software License (OSL 3.0)
+  * that is bundled with this package in the file LICENSE.md.
+  * It is also available through the world-wide-web at this URL:
+  * https://opensource.org/licenses/OSL-3.0
+  * If you did not receive a copy of the license and are unable to
+  * obtain it through the world-wide-web, please send an email
+  * to license@prestashop.com so we can send you a copy immediately.
+  *
+  * DISCLAIMER
+  *
+  * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+  * versions in the future. If you wish to customize PrestaShop for your
+  * needs please refer to https://devdocs.prestashop.com/ for more information.
+  *
+  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+  * @copyright Since 2007 PrestaShop SA and Contributors
+  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+  *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Advparameters.Feature" %}
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{% block content %}
+  <div class="row justify-content-center">
+    {{ form_start(generalForm, {attr : {class: 'form col-xl-10'}, action: path('admin_security_general_save') }) }}
```

Comment:
The form integration is quite "old school" as well In recent form we try to reduce the twig template at the minimum by relying on more detailed FormType classes and using the recent prestashop-ui form_theme

This way almost everything should be rendered by a simple `{{ form_row(generalForm) }}`

Reply:
Should be better right now